gtk4/src/constant_expression.rs

```diff
@@ -11,6 +11,7 @@ define_expression!(
 
 impl ConstantExpression {
     #[doc(alias = "gtk_constant_expression_new")]
+    #[doc(alias = "gtk_constant_expression_new_for_value")]
```

Comment:
new_for_value is already used on the second function `for_value`, is this one needed here too?

Reply:
The all knowing script says so. :)